tests/persons_tests.py

```diff
@@ -4,7 +4,21 @@
 class TestPersons(TestCase):
     def test_add_person_successfully(self):
         new_person = Persons()
```

Comment:
Am curious about your expected implementation of adding Persons -- Fellows or Staff. From this test, it seems that a class called `Persons` has the ability to add Fellows or Staff. To me it doesn't seem natural. I would rather have methods within the `Dojo` class that do the management of Fellows or Staff  as well  as other things like Rooms. 
 What do you think about that?